web/src/pages/project/ProjectGrid.tsx

```diff
@@ -300,9 +306,7 @@ const ProjectGrid: React.FunctionComponent<ProjectGridProps> = ({
                                 // const border = '1px solid'
                                 // debugger
                                 return (
-                                    <SUITable.Row
-                                        key={`${p.external_id}-${s.id}-${sg.id}-${assay.id}`}
-                                    >
+                                    <SUITable.Row key={`${p.id}-${s.id}-${sg.id}-${assay.id}`}>
```

Comment:
Does this need any corresponding updates elsewhere (e.g. in code that tries to link to this row key)?

Reply:
Na, [keys](https://legacy.reactjs.org/docs/lists-and-keys.html#keys) are just for react to tell what's changed between renders

Reply:
Excellent.